src/include/kdbio.h

```diff
@@ -0,0 +1,241 @@
+/**
+ * @file
+ *
+ * @brief
```

Comment:
Purpose of this header file? Is this public?

Reply:
Yes. It contains the ElektraIoInterface which is used by bindings, plugins and applications. I'll add a description.